scripts/translate_schemas.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+current_dir=$(dirname "${BASH_SOURCE[0]}")
+parent_dir=$(dirname "${current_dir}")
+path_to_parent="$( cd "${parent_dir}" && pwd )"
+path_to_schemas="${path_to_parent}"/app/data
+
+# Clone translation schema
+git clone https://github.com/ONSdigital/eq-translations.git "${path_to_parent}"/temp/eq-translations
+"${path_to_parent}"/temp/eq-translations/scripts/run_translate_all_surveys.sh "${path_to_schemas}"
+# Delete repo
+rm -rf "${path_to_parent}"/temp
```

Comment:
I think it would be clearer assigning the temp directory to a variable, something like this:
```
#!/usr/bin/env bash
current_dir=$(dirname "${BASH_SOURCE[0]}")
parent_dir=$(dirname "${current_dir}")
path_to_parent="$( cd "${parent_dir}" && pwd )"

temp_dir="${path_to_parent}"/temp
schemas_dir="${path_to_parent}"/app/data

# Clone translation schema
git clone https://github.com/ONSdigital/eq-translations.git "${temp_dir}"/eq-translations
"${temp_dir}"/eq-translations/scripts/run_translate_all_surveys.sh "${schemas_dir}"
# Delete repo
rm -rf "${temp_dir}"
```